README.md

```diff
@@ -1,6 +1,8 @@
 sbt-doge
 ========
 
+:bangbang: **NOTE: Merged into sbt 1!** This plugin has been merged in sbt 1 and most probably is longer necessary. For further detail see [#18](https://github.com/sbt/sbt-doge/issues/18). :bangbang:
```

Comment:
How about:

> Our favorite sbt-doge has been merged in sbt 1, and thus it will ride into the sunset (The plugin is now retired).

Reply:
mixing horse and dog metaphors... or is it riding a unicorn?

agree this is better wording — the “most probably” hedge just sows unnecessary doubt. ONWARD